discretisedfield/field.py

```diff
@@ -41,8 +41,8 @@ class Field(_FieldIO):
 
     nvdim : int
 
-        Dimension of the field's value. For instance, if `dim=3` the field is a
-        three-dimensional vector field and for `dim=1` the field is a scalar
+        Number of Value DIMensions of the field. For instance, if `nvdim=3` the field is a
```

Comment:
Is DIM capitalised on purpose?
Same with Value

Reply:
Yes - the idea is that the capitalised letters form the argument name (`nvdim`) together.